src/config/css-variables.js

```diff
@@ -3,9 +3,19 @@ const styles = {}
 styles['font-default'] = "'Source Sans Pro', sans-serif"
 styles['font-header'] = "'Poppins', sans-serif"
 
+styles['color-text'] = '#000'
```

Comment:
We need to define the color pallette, remember that we've talked about the color variable names like `color-primary, color-primary-dark ` and so on